optihood/entities.py

```diff
@@ -11,3 +11,15 @@ class NodeKeys(_enum.StrEnum):
     storages = "storages"
     stratified_storage = "stratified_storage"
     profiles = "profiles"
+
+
+class CsvInputFilePathsRelative(_enum.StrEnum):
```

Comment:
This enum ensures we can easily change these entries here, without needing to change it in the rest of the code.